.github/workflows/pull-request.yaml

```diff
@@ -128,6 +128,18 @@ jobs:
         run: make check
         timeout-minutes: 30
 
+  yaml-lint:
+    name: YAML Lint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install yamllint
+        run: pip install yamllint
+
+      - name: Lint YAML files
+        run: yamllint ./test/cases/testdata/
```

Comment:
Question: This is the recommended way to integrate `yamllint`, [according to their docs](https://yamllint.readthedocs.io/en/stable/integration.html#integration-with-github-actions). However, we also have the "do it in Docker" Makefile target, `check-yaml-tests`.

Should I be using the Makefile target here for consistency, or is this "raw" way of running `yamllint` acceptable here? If a comment noting the reason is all we need, I can add that easily enough.

Reply:
The Makefile change pins a version, this "raw way" will just pick the latest. That _could_ be annoying at some point -- but I don't expect either yaml or yamllint to change much over time, so it probably doesn't matter much. I think I'd still lean towards consistency, let's use the make target here 👍

Reply:
Should this be updated to use `make check-yaml-tests` now?

Reply:
Yep! Let me update that now...